app/views/presentations/new.html.erb

```diff
@@ -1,3 +1,3 @@
 <h1>New Presentation</h1>
 
-<%= render 'form' %>
+<%= render partial: 'form', locals: { presentation: @presentation } %>
```

Comment:
This is older rails syntax. You can pass locals in without the keyword args.

`<%= render 'form', presentation: @presentation %>`